scripted_render_pipeline/postcorrector/post_corrector.py

```diff
@@ -0,0 +1,254 @@
+import logging
+
+import numpy as np
+import tifffile
+
+import concurrent.futures
+import logging
+
+import numpy as np
+from tqdm import tqdm
+
+from PIL import Image
```

Comment:
why do we use both PIL and tifffile for the same function? we only need tifffile